src/container/RelayRenderer.js

```diff
@@ -123,10 +124,17 @@ class RelayRenderer extends React.Component {
     super(props, context);
     this.mounted = true;
     this.state = this._runQueries(this.props);
+
+    this.props.relayContext.injectBatchingStrategy(
+      ReactDOM.unstable_batchedUpdates
+    );
```

Comment:
I think it might be good for Relay to have a sensible default here... Maybe we could introduce a `BatchingStrategyLayer` that by default injects `ReactDOM.unstable_batchedUpdates` on the client side, etc. Then you can override this if you'd like.

(In this particular case, the change was made to keep the same behavior as `master`.)